### DISCUSSION TITLE
Add customized field development guide

### DISCUSSION THREAD

**Discussion by dhb52** [orchidsoftware/platform#1524] [Q&A] [answered]
Pls add customized field development guide, Hotwire and Stimulus would be appreciated. 
Without the knowledge of them, the learning curve is quite steep when creating a new Filed.

**Comment by tabuna** [comment, marked as answer]
Hey. We do not require users to know `stimulus/turbolinks/turbo/hotwire`. This is what is convenient for us to use. Users can connect their own libraries or frameworks, for example, popular in the laravel community `Vue`. Therefore, at the moment, we have a description of how to create your own field using only `HTML`. https://orchid.software/en/docs/builder/#representation-of-elements-of-a-form

What do you think about this? Can you suggest an improvement?

**Reply by dhb52** [reply]
That content goes too deep, LOL.

Maybe adding a link for `Customized Filed` on section `Form elements` would be helpful.

Well, I think I was too hasty to raise question here, I should've take more time on RTFM.
